.circleci/config.yml

```diff
@@ -401,6 +401,9 @@ jobs:
       - store_artifacts:
           path: tdrs-frontend/pa11y-screenshots/data-files.png
 
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/welcome.png
```

Comment:
I think we could pass the whole directory to `path` here and not have to define every PNG.